src/Rings/mpoly.jl

```diff
@@ -512,11 +512,12 @@ singular_poly_ring(R::Singular.PolyRing; keep_ordering::Bool = true) = R
 function singular_coeff_ring(K::AbsSimpleNumField)
   minpoly = defining_polynomial(K)
   Qa = parent(minpoly)
-  a = gen(Qa)
   SQa, (Sa,) = Singular.FunctionField(Singular.QQ, _variables_for_singular(symbols(Qa)))
   Sminpoly = SQa(coeff(minpoly, 0))
+  var = Sa
   for i in 1:degree(minpoly)
-    Sminpoly += SQa(coeff(minpoly, i))*Sa^i
+      Sminpoly = muladd(SQa(coeff(minpoly, i)), var, Sminpoly)
```

Comment:
```suggestion
      Sminpoly = addmul!(Sminpoly, SQa(coeff(minpoly, i)), var)
```
Can you check if this works? (`muladd`is a numerial juliaism and is not a useful concept in Oscar land.)

Reply:
`muladd` was an improvement, but after testing `addmul!` is even faster and has fewer allocations